tests/pytests/unit/modules/test_saltutil.py

```diff
@@ -136,3 +156,42 @@ def test_sync_all_clean_pillar_cache():
         with patch("salt.modules.saltutil.refresh_pillar") as refresh_pillar:
             saltutil.sync_all(clean_pillar_cache=True)
             refresh_pillar.assert_called_with(clean_cache=True)
+
+
+def test_list_extmods(salt_call_cli, minion_opts):
+    pathlib.Path(minion_opts["cachedir"], "extmods", "dummydir").mkdir(
+        parents=True, exist_ok=True
+    )
+    ret = saltutil.list_extmods()
+    assert "dummydir" in ret
+    assert ret["dummydir"] == []
+
+
+def test_refresh_beacons():
+    ret = saltutil.refresh_beacons()
+    assert ret is False
+
+
+def test_refresh_matchers():
+    ret = saltutil.refresh_matchers()
+    assert ret is False
+
+
+def test_refresh_modules_async_false():
+    kwargs = {"async": False}
+    ret = saltutil.refresh_modules(**kwargs)
+    assert ret is False
+
+
+def test_clear_job_cache(salt_call_cli, minion_opts):
+    pathlib.Path(minion_opts["cachedir"], "minion_jobs", "dummydir").mkdir(
+        parents=True, exist_ok=True
+    )
+    ret = saltutil.clear_job_cache(hours=1)
+    assert ret is True
+
+
+@pytest.mark.destructive_test
+def test_regen_keys(salt_call_cli, minion_opts):
+    pathlib.Path(minion_opts["pki_dir"], "dummydir").mkdir(parents=True, exist_ok=True)
+    saltutil.regen_keys()
```

Comment:
Are you asserting anything here? Is there anything to assert?

Reply:
Same question

Reply:
If there is a problem, it will throw an error, so how do we test for 'no assertion', let it run, and if there is one, the test will fail.

Reply:
Meh.